t/24-worker-job.t

```diff
@@ -149,7 +155,11 @@ $job->{_status} = 'accepted';
 $job->{_id} = 2;
 
 # try to start job again pretenting isotovideo could actually be spawned
-$engine_mock->mock(engine_workit => sub { {child => $isotovideo} });
+$engine_mock->mock(
+    engine_workit => sub {
+        note('pretenting to run isotovideo');
```

Comment:
same english error as above but I guess you don't even need the note. probably just `$engine_mock->mock(engine_workit => {child => $isotovideo})` is enough

Reply:
I'm going to fix the typos (also in the comments) but I'd like to keep that messages because they make it easier to follow the test and are only printed with `-v` flag anyways.